### PR TITLE
test: improve fungible module coverage

### PR DESCRIPTION
**Pull request by skosito**
# Description

Bumps fungible module coverage from 80 to 90%. Coverage was already improved before (https://github.com/zeta-chain/node/issues/1742), but this improves it a bit more. This is also last issue in the epic https://github.com/zeta-chain/node/issues/1739 so epic can also be closed when this is merged.

Closes: #1977 

## Type of change

Tests

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [x] Go unit tests
- [ ] Go integration tests
- [x] Tested via GitHub Actions 

# Checklist:

- [x] I have added unit tests that prove my fix feature works

### CODE REVIEW

File: x/fungible/keeper/deposits_test.go [hunk 1]
```diff
@@ -352,3 +353,17 @@ func TestKeeper_ZRC20DepositAndCallContract(t *testing.T) {
...
+	k, ctx, _, _ := testkeeper.FungibleKeeper(t)
+	to := sample.EthAddress()
+	amount := big.NewInt(1)
+	zetaToAddress := sdk.AccAddress(to.Bytes())
+
+	b := k.GetBankKeeper().GetBalance(ctx, zetaToAddress, "azeta")
+	require.Equal(t, int64(0), b.Amount.Int64())
+
+	k.DepositCoinZeta(ctx, to, amount)
+	b = k.GetBankKeeper().GetBalance(ctx, zetaToAddress, "azeta")
```
Comment (kingpinXD) [inline x/fungible/keeper/deposits_test.go:367]: NIT : we can use config.BaseDenom

Comment (kingpinXD) [inline x/fungible/keeper/deposits_test.go:367]: Can't we use sdk keepers and not have to add GetBalance to the expected keepers?

File: x/fungible/types/expected_keepers.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ type AccountKeeper interface {
 type BankKeeper interface {
 	SendCoinsFromModuleToAccount(ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
 	MintCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
+	GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin
```
Comment (kingpinXD) [inline x/fungible/types/expected_keepers.go:31]: related to my comments on deposits_test.go , is this needed ?

**Review by kingpinXD — COMMENTED** [review]
lgtm , left a few minor comments

File: x/fungible/keeper/deposits_test.go [hunk 3]
```diff
@@ -352,3 +353,17 @@ func TestKeeper_ZRC20DepositAndCallContract(t *testing.T) {
...
+func TestKeeper_DepositCoinZeta(t *testing.T) {
+	k, ctx, _, _ := testkeeper.FungibleKeeper(t)
+	to := sample.EthAddress()
+	amount := big.NewInt(1)
+	zetaToAddress := sdk.AccAddress(to.Bytes())
+
+	b := k.GetBankKeeper().GetBalance(ctx, zetaToAddress, "azeta")
+	require.Equal(t, int64(0), b.Amount.Int64())
+
+	k.DepositCoinZeta(ctx, to, amount)
```
Comment (kingpinXD) [inline x/fungible/keeper/deposits_test.go:366]: maybe we can check the error for this call

**Review by kingpinXD — APPROVED** [review]

**Review by lumtis — APPROVED** [review]
👍

**Pull request merged**